src/frontend/qt_sdl/main.cpp

```diff
@@ -99,6 +101,51 @@
 
 // TODO: uniform variable spelling
 
+const QStringList NdsRomMimeTypes { "application/x-nintendo-ds-rom" };
+const QStringList NdsRomExtensions { ".nds", ".srl", ".dsi", ".ids" };
+
+const QStringList GbaRomMimeTypes { "application/x-gba-rom" };
+const QStringList GbaRomExtensions { ".gba", ".agb" };
+
+const QStringList ArchiveMimeTypes {
```

Comment:
{ on the next line

Reply:
Fixed in 04061eda0b08c0f51428fb4dc93cc2f7e8f37686.